recipes/tardis/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "tardis-sn" %}
+{% set version = "3.0.a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+  host:
+    - python
+    - pip
+    - astropy
+    - setuptools
+    - Cython =0.29
+
+  run:
+    - python >=3
```

You shouldn't constrain the Python version unless it's a `noarch` package (which means the same "build" can be used on Windows, macOS and Linux, such as a pure Python library).
```suggestion
    - python
```
The constraint will come from the `skip: True  # [py2k]` above.

---

Also you don't need to put the `[skip appveyor]` in all your commit messages. Now the Windows build is skipped correctly the CI will just pass as it has nothing to do.